yarn.lock

```diff
@@ -2663,11 +2663,30 @@
     "@sentry/utils" "6.19.6"
     tslib "^1.9.3"
 
+<<<<<<< Updated upstream
 "@sentry/types@6.19.6", "@sentry/types@^6.11.0":
+=======
+<<<<<<< HEAD
+"@sentry/types@6.19.6":
```

Comment:
What package updates were needed in this file? Why were they needed?

Reply:
Sorry, i didn't pay full attention to the yarn.lock, i didn't introduce any new deps, that's probably due to my branch not being up to date with master, i'll get it fixed.